applications/nrf_desktop/src/events/passkey_event.c

```diff
@@ -55,12 +58,15 @@ static void profile_passkey_req_event(struct log_event_buf *buf,
 	profiler_log_encode_uint8(buf, event->active);
 }
 
+const uint8_t passkey_req_event_flags = (IS_ENABLED(CONFIG_DESKTOP_INIT_LOG_PASSKEY_EVENT))
```

Comment:
I thought that in such way it will be not readable when you will have more flags which you want to send. I also asked you to give such review earlier before I used 2h to manually change EVENT_TYPE_DEFINE (lots of different ways and it was faster to do manually). If you will insist I can change it to the way you suggest.

Reply:
You can always get rid of the variable and still allow compile time optimisation - use enum value. Look how it was done in my PR: https://github.com/nrfconnect/sdk-nrf/pull/6798/files#diff-116eb9ddb99952b4981b5619fe0f0a31a5493fdeb6301175e4ff269343269866R184. Line 185 in event_manager_priv.h. From the top of my mind the previous version I commented, I was sure the way you are implementing it is there not to change the event definition API. Now as it was changed anyway - it does no more make a big sense to fight with it.

Reply:
I have to change TYPE_DEFINE api so that it can easily take some more flags if needed. 

Do I understand correctly that it is ok for you to leave it as it is with flag variable?